SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +25,29 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+Requires:       kernel
 Requires:       python3
 
+# set package name
+%{!?_name: %global _name mstflint}
+
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global install_mod_dir extra/%{_name}
+
 %description
 This package contains firmware update tool, vpd dump and register dump tools
 for network adapters based on Mellanox Technologies chips.
 
+%package kernel
+Summary:        mstflint Kernel Modules
+Group:          System Environment/Kernel
+Requires:       kernel-headers >= 5.15.0
```

Comment:
Is this true? Does a built kernel module need any headers? And if it does, please specify the exact version with `=` instead of `>=`.

Reply:
yes, we are limiting this module to kernel 5.15.x